kickstart/part/h4113

```diff
@@ -1,3 +1,4 @@
 part / --fstype="ext4" --size=1800 --label=root
 part /home --fstype="ext4" --size=800 --label=home
 part /fimage --fstype="ext4" --size=10 --label=fimage
+part /opt/alien --fstype="ext4" --size=850 --label=alien
```

Comment:
Given that this is correct I can approve this.

Reply:
Good question, factory reset does not work...